app/models/batch_import.rb

```diff
@@ -152,10 +152,10 @@ def ensure_imports_are_complete!
       imports_in_progress = import_count('in_progress').positive?
 
       if cancelled && imports_in_progress
-        errors[:base] << 'Cannot destroy cancelled batch import while imports are in_progress'
+        errors.add(:base, 'Cannot destroy cancelled batch import while imports are in_progress')
```

Comment:
https://edgeapi.rubyonrails.org/classes/ActiveModel/Errors.html suggests that this should be `errors.add(:base, message: 'Cannot destroy cancelled batch import while imports are in_progress')` ... does it work as expected with the two parameters?

Reply:
It does.  In the `add` method description here (https://edgeapi.rubyonrails.org/classes/ActiveModel/Errors.html#method-i-add) it says `If type is a string, it will be used as error message.`.